Canvas/GradesWidget/Base.lproj/MainInterface.storyboard

```diff
@@ -18,24 +18,34 @@
                         <rect key="frame" x="0.0" y="0.0" width="320" height="500"/>
                         <autoresizingMask key="autoresizingMask" widthSizable="YES" heightSizable="YES"/>
                         <subviews>
-                            <tableView clipsSubviews="YES" contentMode="scaleToFill" preservesSuperviewLayoutMargins="YES" alwaysBounceVertical="YES" scrollEnabled="NO" dataMode="prototypes" style="plain" separatorStyle="default" rowHeight="-1" estimatedRowHeight="-1" sectionHeaderHeight="28" sectionFooterHeight="28" translatesAutoresizingMaskIntoConstraints="NO" id="5Mh-TY-EQc">
+                            <tableView clipsSubviews="YES" contentMode="scaleToFill" preservesSuperviewLayoutMargins="YES" alwaysBounceVertical="YES" dataMode="prototypes" style="plain" separatorStyle="default" rowHeight="-1" estimatedRowHeight="-1" sectionHeaderHeight="28" sectionFooterHeight="28" translatesAutoresizingMaskIntoConstraints="NO" id="5Mh-TY-EQc">
                                 <rect key="frame" x="0.0" y="20" width="320" height="480"/>
                                 <color key="backgroundColor" white="1" alpha="1" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
                                 <color key="sectionIndexBackgroundColor" white="0.0" alpha="0.0" colorSpace="custom" customColorSpace="genericGamma22GrayColorSpace"/>
                             </tableView>
+                            <button hidden="YES" opaque="NO" contentMode="scaleToFill" contentHorizontalAlignment="center" contentVerticalAlignment="center" buttonType="roundedRect" lineBreakMode="middleTruncation" translatesAutoresizingMaskIntoConstraints="NO" id="iAh-JB-PVm" customClass="DynamicButton" customModule="Core">
+                                <rect key="frame" x="16" y="454" width="73" height="30"/>
+                                <state key="normal" title="View More"/>
+                                <connections>
+                                    <action selector="openApp:" destination="3tg-Or-QnS" eventType="touchUpInside" id="iYG-BH-nRq"/>
```

Comment:
should we use the primary action, instead of touch up inside?

Reply:
Yeah I can do that. I just did the ctrl drag in storyboard and whatever it hooked up. I will get that changed.